guardian/ArchiveHunterDownloadManager
# Implement Bulk Version Two Protocol

## What does this change?

Implements the use of the bulk version two protocol.

## How to test

Build the executable with Xcode and try to download a bulk from the server. The bulk should download successfully.

## How can we measure success?

Bulks with many files should now download successfully.
